src/PhpParser/NodeTraverser/RectorNodeTraverser.php

```diff
@@ -60,6 +93,7 @@ private function prepareNodeVisitors(): void
 
         // filer out by version
         $this->visitors = $this->phpVersionedFilter->filter($this->rectors);
+        $this->visitorsPerNodeClass = [];
```

Comment:
can initialization be moved to property as default empty array?

```php
    private array $visitorsPerNodeClass = [];
```

Reply:
I thought I could not do this because we need to reset the cache between tests but the intialisation in `refreshPhpRectors()` is enough, so I changed this